matrix_vision.py

```diff
@@ -1,14 +1,57 @@
-# import numpy as np
+import imageio
+import pygame as pg
+import random
 
 
-class Matrix:
-    def __init__(self):
-        pass
+class MatrixLetters:
+    def __init__(self, width, height):
+        self.width = width
+        self.height = height
+        self.letters = [chr(int('0x30a0', 16) + i) for i in range(1, 95)]
+        self.font_size = 15
+        self.font = pg.font.Font('ms_mincho.ttf', self.font_size, bold=True)
```

Comment:
I suggest using 
self.font = pg.font.SysFont('ms mincho', self.font_size, bold=True)
In this implementation, it will use system fonts